bin/run.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# default variables
+: "${PORT:=8080}"
+: "${SLEEP:=1}"
+: "${TRIES:=60}"
+
+usage() {
+  echo "usage: ./bin/run.sh web|test|test-ci|bash"
+  exit 1
+}
+
+wait_for() {
+  tries=0
+  echo "Waiting for $1 to listen on $2..."
+  while true; do
+    [[ $tries -lt $TRIES ]] || return
+    (echo > /dev/tcp/$1/$2) >/dev/null 2>&1
+    result=
+    [[ $? -eq 0 ]] && return
+    sleep $SLEEP
+    tries=$((tries + 1))
+  done
+}
+
+[ $# -lt 1 ] && usage
+
+# Only wait for backend services in development
+# http://stackoverflow.com/a/13864829
+# For example, bin/test.sh sets 'DEVELOPMENT' to something
+[ ! -z ${DEVELOPMENT+check} ] && wait_for db 5432 && sleep 3
+
+case $1 in
+  web)
+    /go/bin/tigerblood "${@:2}"
+    ;;
+  test)
+    cd /go/src/go.mozilla.org/tigerblood
+    go test -v
```

Comment:
Might want to test ./... in case subpackages are added in the future

Reply:
like https://github.com/mozilla-services/tigerblood/pull/89/commits/ce917b4ecb34966c2e405dfe2ec54cf2f8257ac5 i.e. `go test -v ./` or a different path?